player/public/coreplugins/org.sunbird.assess.endpage-1.0/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "id": "org.sunbird.assess.endpage",
+    "ver": "1.0",
+    "shortId": "org.sunbird.assess.endpage",
+    "author": "Jagadish Pujari",
+    "description": "",
+    "publishedDate": "",
+    "renderer": {
+        "main": "renderer/plugin.js",
+        "dependencies": [
+            {"type": "css", "src": "renderer/templates/style.css"}
+        ]
+    },
+    "dependencies": [
+        {"plugin": "org.sunbird.player.userswitcher", "ver": "1.0", "scope": "renderer" }
```

Comment:
Remove this user-switcher dependency here. Not required for assessment or exam contents